EnglishTranslation-master/ETC.tsv

```diff
@@ -25942,7 +25942,7 @@ ETC_20161214_025936	Soul Crystal (14 Days)	소울 크리스탈(14일)
 ETC_20161214_025937	Using this Soul Crystal will resurrect you in the same position you died. Upon pick-up, use within 14 days.	전투 불능 시 소울 크리스탈을 사용하여 제자리에서 부활할 수 있습니다. 획득 후부터 14일동안 사용 가능합니다.				
 ETC_20161214_025938	EXP Tome: 10 Minutes	경험의 서:10분				
 ETC_20161214_025939	Applies various effects to your character for 1 hour. Upon pick-up, use within 10 minutes. Right-click to use. Upon pick-up, use within 14 days.{nl} - Consumes time when you are connected to the game.{nl}[EXP Tome Effect]{nl} - +30% EXP from hunting monsters{nl} - Doubled STA recovery rate	마우스 우클릭 시 사용되며 캐릭터에 1시간 동안 효과가 적용됩니다. 획득 후부터 10분 동안 사용 가능합니다. 획득 후부터 14일동안 사용 가능합니다.{nl} - 캐릭터 접속 시에만 시간이 소진됩니다.{nl}[경험의 서 효과]{nl} - 몬스터 처치 시 획득 경험치 +30% 증가{nl}(PC방 혜택, 토큰 효과와 중첩 가능){nl}(경험치 카드 적용안됨){nl} - 스태미나 회복량 2배 증가				
-ETC_20161214_025940	Instance One Entry Voucher	인스턴스 던전 1회 입장권				
+ETC_20161214_025940	Instanced One Entry Voucher	인스턴스 던전 1회 입장권				
```

Comment:
Why instanced? It's +1 instance entry voucher.

Reply:
`Instanced One Entry Voucher` -> `Instanced Dungeon One Entry Voucher`